### DISCUSSION TITLE
Can't create new projects for my account

### DISCUSSION THREAD

**Discussion by tusharsable** [iTwin/itwinjs-core#2817] [Q&A]
Hi All we were trying to create a project on https://connect.bentley.com/ by following the tutorial video [here](https://learn.bentley.com/app/VideoPlayer/LinkToIndividualCourse?LearningPathID=109270&CourseId=114330&MediaID=5006537)
![image](https://user-images.githubusercontent.com/66006244/144417750-81b40168-765b-4fbe-b72c-fad39e3b43f7.png)
We couldn't find such a button in my account to create a new project.
![image](https://user-images.githubusercontent.com/66006244/144417908-630f1a98-d330-40b2-96c6-a2fe1aa6541a.png)

Also we observed that I am not a administrator of account create by me on bentley. Maybe That could be one reason.

**Comment by JohnMajerle** [comment]
Creation of the project doesn't make you a member of the project.  To do that you must go to the project page and then select **_Manage_** under Team Members.  A for will be displayed that will let you add members to the project.
